From b62cf2d658a90e3c27edc561a030d1a8acbbed39 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 28 Sep 2012 11:17:18 +0800
Subject: ENGR00225960-01 Sii902x HDMI: added SII902x Config variable

-Add CONFIG_FB_MXC_SII902x_ELCDIF variable to mx6s config files.
-Default enable CONFIG_FB_MXC_ELCDIF_FB,
CONFIG_FB_MXC_SEIKO_WVGA_SYNC_PANEL
and CONFIG_FB_MXC_SII902x_ELCDIF.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/configs/imx6s_defconfig |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6s_defconfig b/arch/arm/configs/imx6s_defconfig
index 95e07d5..ecb8ee88 100644
--- a/arch/arm/configs/imx6s_defconfig
+++ b/arch/arm/configs/imx6s_defconfig
@@ -1721,15 +1721,16 @@ CONFIG_FB_MXC_LDB=y
 CONFIG_FB_MXC_MIPI_DSI=y
 CONFIG_FB_MXC_TRULY_WVGA_SYNC_PANEL=y
 # CONFIG_FB_MXC_CLAA_WVGA_SYNC_PANEL is not set
-# CONFIG_FB_MXC_SEIKO_WVGA_SYNC_PANEL is not set
+CONFIG_FB_MXC_SEIKO_WVGA_SYNC_PANEL=y
 # CONFIG_FB_MXC_SII902X is not set
+CONFIG_FB_MXC_SII902X_ELCDIF=y
 # CONFIG_FB_MXC_CH7026 is not set
 # CONFIG_FB_MXC_TVOUT_CH7024 is not set
 # CONFIG_FB_MXC_ASYNC_PANEL is not set
 CONFIG_FB_MXC_EINK_PANEL=y
 # CONFIG_FB_MXC_EINK_AUTO_UPDATE_MODE is not set
 # CONFIG_FB_MXC_SIPIX_PANEL is not set
-# CONFIG_FB_MXC_ELCDIF_FB is not set
+CONFIG_FB_MXC_ELCDIF_FB=y
 CONFIG_FB_MXC_HDMI=y
 
 #
-- 
1.7.9.5

